core/storage/statistics/gae_models.py

```diff
@@ -973,14 +973,14 @@ class PlaythroughModel(base_models.BaseModel):
     exp_id = ndb.StringProperty(indexed=True)
     # Version of the exploration.
     exp_version = ndb.IntegerProperty(indexed=True)
-    # ID of the issue.
-    issue_id = ndb.StringProperty(indexed=True)
-    # The customization args dict for the given issue_id.
+    # Type of the issue.
+    issue_type = ndb.StringProperty(indexed=True)
+    # The customization args dict for the given issue_type.
     issue_customization_args = ndb.JsonProperty(default=None)
     # The playthrough actions for this playthrough. This will be a list of dicts
     # where each dict represents a single playthrough action. The list is
     # ordered by the time of occurence of the action.
-    playthrough_actions = ndb.JsonProperty(default=None)
+    playthrough_actions = ndb.JsonProperty(default=None, repeated=True)
```

Comment:
I think if repeated is specified, you don't need default -- the default value will just be []. Having, additionally, a default of None seems confusing.

(Please double-check the relevant App Engine docs on ndb datastore properties before resolving this.)

Reply:
Done.